src/afl-fuzz-stats.c

```diff
@@ -881,7 +888,7 @@ void show_stats(afl_state_t *afl) {
                 "    levels : " cRST "%-10s" bSTG       bV "\n",
        tmp, u_stringify_int(IB(0), afl->max_depth));
 
-  if (!afl->skip_deterministic) {
+  if (unlikely(!afl->skip_deterministic)) {
```

Comment:
Still default for `-M`. I would leave this decision to the compiler to optimize, maybe

Reply:
yes you would never run -M without at least one -S or usually more -S, so this is better.